thirdparty.js

```diff
@@ -28,7 +29,7 @@ var Thirdparty = {}
  *
  * FIXME: not optimised at all
  */
-function evp_kdf (data, salt, opts) {
+function evpKdf (data, salt, opts) {
```

Comment:
This seems to be an irrelevant change.

Reply:
This fixes linting.

Reply:
Since when does it have a rule of not using underscores? Seems very weird.

Reply:
Names have to be ``camelCase``. My favorite rule for lint-hating is actually this trailing-spaces rule. :-)

Reply:
@axic I've now un-done the name change from above to avoid side effects (unlikely though). This should now ready to be merged.